.github/workflows/wine-arch.yml

```diff
@@ -0,0 +1,29 @@
+name: Wine Archlinux CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: archlinux:latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Compilation
+        run: |
+          echo -e "[multilib]\nInclude = /etc/pacman.d/mirrorlist" >> /etc/pacman.conf
+          yes|pacman -Syu --noconfirm base-devel sudo
```

Comment:
Are you sure `sudo` is necessary, even if we use `su` later on?

Reply:
Build scripts required it.

Reply:
Ah, I didn't think about that ^^